com.reprezen.swagedit.openapi3/src/com/reprezen/swagedit/openapi3/assist/ext/SchemaTypeContentAssistExt.java

```diff
@@ -0,0 +1,33 @@
+package com.reprezen.swagedit.openapi3.assist.ext;
+
+import java.util.Collection;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.google.common.collect.Lists;
+import com.reprezen.swagedit.core.assist.Proposal;
+import com.reprezen.swagedit.core.assist.ext.ContentAssistExt;
+import com.reprezen.swagedit.core.model.AbstractNode;
+import com.reprezen.swagedit.core.schema.TypeDefinition;
+
+public class SchemaTypeContentAssistExt implements ContentAssistExt {
+
+    private static final JsonPointer pointer = JsonPointer.compile("/definitions/schema/properties/type");
+
+    @Override
+    public boolean canProvideContentAssist(TypeDefinition type) {
+        return type != null && pointer.equals(type.getPointer());
+    }
+
+    @Override
+    public Collection<Proposal> getProposals(TypeDefinition type, AbstractNode node, String prefix) {
+        return Lists.newArrayList( //
+                new Proposal("array", "array", null, "enum"), //
+                new Proposal("boolean", "boolean", null, "enum"), //
+                new Proposal("integer", "integer", null, "enum"), //
+                new Proposal("null", "null", null, "enum"), //
```

Comment:
As per SUM discussion, we use a `"null"` string (quoted) instead of a `null` value, see `com.reprezen.swagedit.core.assist.JsonProposalProvider.createPropertyProposal(String, TypeDefinition)`
```java
        if (key == "null") {
            key = "\"" + key + "\"";
        }
```
We even added a validation rule for the null type in Swagger v2, see https://github.com/RepreZen/KaiZen-OpenAPI-Editor/pull/263